tests/conftest.py

```diff
@@ -1,34 +1,74 @@
 from pathlib import Path
 import pytest, os, uuid, shutil
+from typing import List
 
-from sand.models.base import init_db
+from sand.models.base import db, init_db
+from sand.models import all_tables
+from werkzeug.datastructures import FileStorage
+from sand.controllers.helpers.upload import (
+    CSVParserOpts,
+    UploadingTable,
+    get_extension,
+    parse_upload,
+    save_upload,
+)
+import json
 
 
-@pytest.fixture()
+@pytest.fixture
 def client():
-    tempdir = Path("/tmp") / str(uuid.uuid4())
-
     try:
-        tempdir.mkdir()
-        shutil.copyfile(
-            Path(__file__).parent / "resources/sand.db", str(Path(tempdir) / "sand.db")
-        )
-        init_db(str(Path(tempdir) / "sand.db"))
+        init_db(":memory:")
+        db.create_tables(all_tables, safe=False)
 
-        from sand.app import app
         from sand.config import SETTINGS, _ROOT_DIR
+        from sand.app import app
+
+        externaldb = _ROOT_DIR / "tests/resources/data/databases"
 
-        externaldb = _ROOT_DIR / "../data/home/databases"
-        for cfg in SETTINGS.values():
+        filtered_setings = {k: v for k, v in SETTINGS.items() if k in ['entity', 'ont_classes', 'ont_props']}
+        for cfg in filtered_setings.values():
             cfg["args"]["dbfile"] = os.path.join(
                 externaldb, Path(cfg["args"]["dbfile"]).name
             )
             assert os.path.exists(cfg["args"]["dbfile"]), cfg["args"]["dbfile"]
             cfg["args"]["proxy"] = False
 
         app.config["TESTING"] = True
+
         with app.test_client() as client:
             yield client
     finally:
-        if tempdir.exists():
-            shutil.rmtree(tempdir, ignore_errors=True)
+        db.drop_tables(all_tables)
+        db.close()
+
+
+@pytest.fixture
+def load_db(client):
```

Comment:
name this example_db, we can have multiple db for different tests in the future.

Reply:
Updated the fixture name